app-backend/migrations/src_migrations/main/scala/73.scala

```diff
@@ -0,0 +1,10 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M73 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(73)(List(
+    sqlu"""
+CREATE INDEX scene_name ON scenes (name);
```

Comment:
I don't know if this really matters, but it seems like the majority of the existing indexes have a `_idx` suffix:

```
Indexes:
    "scenes_pkey" PRIMARY KEY, btree (id)
    "acquisition_date_idx" btree (acquisition_date)
    "cloud_cover_idx" btree (cloud_cover)
    "create_idx" btree (created_at)
    "created_at_idx" btree (created_at)
    "data_footprint_idx" gist (data_footprint)
    "name_idx" btree (name)
    "owner_idx" btree (owner)
    "visibility_idx" btree (visibility)
```